vllm/compilation/partition_rules.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+
+from __future__ import annotations
+
+import contextlib
+
+from torch._library.utils import lookup_op
+
+from vllm.logger import init_logger
+
+logger = init_logger(__name__)
+
+
+def _resolve_operator_overload(op_name: str):
+    """Resolve vLLM operator name to torch.ops OpOverload.
```

Comment:
I would actually rather just use the PyTorch syntax everywhere to simplify the code, I don't think we have to use special syntax.